products/public-websites/Cerner-Support/how-to-move-facility-to-cerner.md

```diff
@@ -0,0 +1,42 @@
+# How to move a non-Cerner facility to being a Cerner facility
+
+## Purpose
+
+If you just heard about a facility that is migrating to Cerner, then this document will walk you through how to prepare for it.
+
+## Steps
+
+For the following steps, we will use __facility ID **463**__ as an example.
+
+1. Obtain the facility ID of the facility that will soon go live as a Cerner facility.
+
+1. Create a Pull Request on [`vets-api`](https://github.com/department-of-veterans-affairs/vets-api/pulls) that adds the facility ID as a feature toggle. For example, for facility ID 463, you should add:
+
+```yml
+cerner_override_463:
+  actor_type: user
+  description: >
+    This will show the Cerner facility 463 as `isCerner`.
+```
+
+An example Pull Request for this can be found [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6045/files).
+
+1. Wait until your `vets-api` Pull Request is merged **and deployed**. You will then be able to find your feature toggles at https://api.va.gov/flipper/features. Refer to [the guide on how to work with Flipper feature toggles](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/) for more details on how to enable/disable/etc. your new feature toggles.
+
+**WARNING: YOU MUST MERGE AND DEPLOY THE `vets-api` PULL REQUEST BEFORE MERGING THE `vets-website` PULL REQUEST DESCRIBED BELOW.**
```

Comment:
Is there anything code-wise we can do to get around this ominous warning? Lol

Reply:
Very likely! We have to do this right now because of the `=== false` [here](https://github.com/department-of-veterans-affairs/vets-website/blob/f2965706ccd1d500c1bc4a25c2a236f9eea6bb3b/src/platform/user/selectors.js#L30). If we could do `!state?.featureToggles?.[`cerner_override_${facilityId}`]` here instead, then it wouldn't be a problem, although I remember @jbalboni @erikphansen you and I decided to go for `=== false` for a reason (which I cannot remember why).